src/interfaces/IMetaMorpho.sol

```diff
@@ -7,8 +7,8 @@ import {IERC4626} from "@openzeppelin/interfaces/IERC4626.sol";
 struct MarketConfig {
     /// @notice The maximum amount of assets that can be allocated to the market.
     uint192 cap;
-    /// @notice The rank of the market in the withdraw queue.
-    uint64 withdrawRank;
+    /// @notice Whether the market is in the withdraw queue.
+    bool inWithdrawQueue;
```

Comment:
```suggestion
    bool enabled;
```
Actually enabled markets are markets in the withdraw queue. Could make sense to make it explicit with this naming

Reply:
Smart! I like it